index.bs

```diff
@@ -560,6 +560,7 @@ spec:csp3; type:grammar; text:base64-value
   1. If |parsedMetadata| is not the empty set and
      |request|'s <a for=request>mode</a> is either "`cors`" or "`same-origin`",
      return "Allowed".
+  1. If |request|'s <a for=request>url</a> is data or blob URL, return "Allowed".
```

Comment:
Maybe `If |request|'s <a for=request>url</a>'s <a for=url>scheme</a> is "data" or "blob", return "Allowed".`?